.gitignore

```diff
@@ -249,4 +249,8 @@ paket-files/
 
 # JetBrains Rider
 .idea/
-*.sln.iml
\ No newline at end of file
+*.sln.iml
+/src/MrEric.R91/Feature/PublicAutoPropertyInitializationContext.cs
+/src/MrEric.R91/Feature/IntroduceAndInitializePublicAutoPropertyExectutor.cs
+/src/MrEric.R91/Feature/Intentions/IntroduceAndInitializePublicAutoPropertyFix.cs
```

Comment:
No idea why they are!..  I did revert the repro several times as I tried different approaches, might be legacy.. will take a look at now.